src/popup/pages/AuthorizedAccounts.vue

```diff
@@ -0,0 +1,159 @@
+            class="account-row"
+          >
+            <Avatar
+              class="avatar"
+              size="sm"
+              :address="address"
+            />
+            <AddressTruncated
+              :address="address"
+              class="ae-address"
+            />
+            <LinkButton :to="getExplorerPath(address)">
+              <ExternalLink class="external-icon" />
+            </LinkButton>
+          </div>
+        </div>
+        <div class="dialog">
+          <div>
+            {{ $t('pages.multisig-details.signers') }}
+            <span class="bold">
+              {{ authorizedCountMock }}
+            </span>
+          </div>
+          <div>
+            {{ $t('pages.multisig-details.approval') }}
+            <span class="bold">
+              {{ needForApprovalCountMock }}
+            </span>
+          </div>
+        </div>
+      </div>
+    </template>
+  </DetailsItem>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from '@vue/composition-api';
+import Avatar from '../components/Avatar.vue';
+import AddressTruncated from '../components/AddressTruncated.vue';
+import ExternalLink from '../../icons/external-link.svg?vue-component';
+import { useGetter } from '../../composables/vuex';
+import LinkButton from '../components/LinkButton.vue';
+import DetailsItem from '../components/DetailsItem.vue';
+
+export default defineComponent({
+  name: 'AuthorizedAccounts',
+  components: {
+    DetailsItem,
+    LinkButton,
+    AddressTruncated,
+    Avatar,
+    ExternalLink,
+  },
+  props: {
+    addressList: {
+      type: Array as PropType<string[]>,
+      default: () => [],
+    },
+  },
+  setup() {
+    const getExplorerPath = useGetter('getExplorerPath');
+
+    // TODO Authorized signers value
+    const authorizedCountMock = 3;
+    // TODO Needed for approval value
+    const needForApprovalCountMock = 3;
+
+    return {
+      getExplorerPath,
+      authorizedCountMock,
+      needForApprovalCountMock,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../styles/variables';
+@use '../../styles/typography';
+
+.authorized-accounts {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+
+  .account-row {
+    display: flex;
+    align-items: center;
+  }
+
+  .avatar {
+    margin-right: 4px;
+  }
+
+  .external-link {
+    margin-left: 7px;
+    width: 22px;
+    height: 22px;
+
+    .external-icon {
+      opacity: 1;
+    }
+  }
+
+  .account-list,
+  .dialog {
+    width: 100%;
+  }
+
+  .account-list {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  .ae-address {
+    color: rgba(variables.$color-white, 0.85);
+  }
+
+  .dialog {
+    @extend %face-sans-12-regular;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: 8px;
+    background-color: variables.$color-grey-dialog;
+    height: 100%;
+    position: relative;
+    padding: 21px 8px;
+
+    .bold {
```

Comment:
`<bold>` and `.bold` are not semantically correct. I suggest to use `<strong>` html tag here.